app/locales/de/messages.po

```diff
@@ -13,19 +13,23 @@ msgstr ""
 "Language-Team: \n"
 "Plural-Forms: \n"
 
+#: app/configurator/components/chart-configurator.tsx:385
+msgid "Add filter"
+msgstr ""
```

Comment:
Missing trnslation ?

Reply:
I have marked this PR as auto merge, but I will handle these translations in another PR, thanks!

Reply:
Solved on main